spec.bs

```diff
@@ -1644,11 +1648,17 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
               |settings|'s [=environment settings object/current monotonic time=], in milliseconds.
             1. If |perBuyerCumulativeTimeout| is not null, then decrement |perBuyerCumulativeTimeout|
               by |generateBidDuration|.
-            1. If |generatedBid| is failure, [=iteration/continue=].
-          1. [=list/Insert=] |generatedBid|'s [=generated bid/interest group=] in |bidIgs|.
-          1. [=Score and rank a bid=] with |auctionConfig|, |generatedBid|, |leadingBidInfo|,
-            |decisionLogicScript|, |directFromSellerSignalsForSeller|, |dataVersion|, |auctionLevel|,
-            |componentAuctionExpectedCurrency|, and |topLevelOrigin|.
+            1. If |generatedBid| is not a failure:

```

Comment:
Hmm, might need to register the original generatedBid for debugging if this one is a failure to get the reject reason  = below threshold stuff out?

Reply:
Though this appears that it may only work for PA, not for forDebuggingOnly?

https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/interest_group_auction.cc;drc=3afa53375890e7144673d8e9e6356868295c2873;l=490

... which may make my pull request to the explainer a bit off :(

Reply:
hmm, seems like a bug, that we may want to report this k-anon reject reason to forDebuggingOnly as well?

Reply:
Yeah, probably. And I think my next refactors of this make us better suited for speccing it, so maybe don't need to worry about this for a moment.